roads.mss

```diff
@@ -3100,3 +3100,81 @@ tertiary is rendered from z10 and is not included in osm_planet_roads. */
     }
   }
 }
+#railways-text-name {
+  /* Others started from z15. */
```

Comment:
I think z18 is a little bit late; anyway I will try to render New York to see how does it look.